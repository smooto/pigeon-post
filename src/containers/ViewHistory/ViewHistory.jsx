import React from 'react';
import PropTypes from 'prop-types';

const ViewHistory = ({ history }) => (
  <section>
    <h2>Request History</h2>
    <div>
      <ul>
        {
          history.map((request, i) => (
            <li key={i}>
              <p>{request.requestType}</p>
              <p>{request.url}</p>
            </li>
          ))
        }
      </ul>
    </div>
  </section>
);

ViewHistory.propTypes = {
  history: PropTypes.array.isRequired
};

export default ViewHistory;
