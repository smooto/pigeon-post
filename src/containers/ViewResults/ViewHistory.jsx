import React from 'react';

const ViewResults = () => (
  <section>
    <h2>Results</h2>
    <div>
      {/* <pre>
        "example": {
          "json" : "here's some example JSON"
        }
      </pre> */}
    </div>
  </section>
);

export default ViewResults;
