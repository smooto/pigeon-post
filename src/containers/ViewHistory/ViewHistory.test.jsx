import React from 'react';
import { shallow } from 'enzyme';
import ViewHistory from './ViewHistory';

describe('ViewHistory component', () => {
  it('renders ViewHistory', () => {
    const wrapper = shallow(<ViewHistory history={[]} />);
    expect(wrapper).toMatchSnapshot();
  });
});
