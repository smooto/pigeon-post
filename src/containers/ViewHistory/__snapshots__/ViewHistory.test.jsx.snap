// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewHistory component renders ViewHistory 1`] = `
<section>
  <h2>
    Request History
  </h2>
  <div>
    <ul />
  </div>
</section>
`;
